arangod/Cache/PlainCache.h

```diff
@@ -46,11 +46,13 @@ namespace arangodb::cache {
 /// simple API following that of the base Cache class. For any non-pure-virtual
 /// functions, see Cache.h for documentation.
 ////////////////////////////////////////////////////////////////////////////////
+
+template<typename Hasher>
 class PlainCache final : public Cache {
  public:
-  PlainCache(Cache::ConstructionGuard guard, Manager* manager, std::uint64_t id,
-             Metadata&& metadata, std::shared_ptr<Table> table,
-             bool enableWindowedStats);
+  PlainCache(Cache::ConstructionGuard guard, Manager* manager, Hasher hasher,
+             std::uint64_t id, Metadata&& metadata,
+             std::shared_ptr<Table> table, bool enableWindowedStats);
   ~PlainCache();
```

Comment:
Do we actually need to store a `Hasher` instance or could we get away with the type? At the moment the two hashers have non-static member functions, but since they are both completely stateless we could make them static and get rid of our `_hasher` member here.

Reply:
Yes, that would be possible. Do you think that is worth it?

Reply:
Removed the hasher instance now.